WalletWasabi.Fluent/Controls/TagsBox.axaml

```diff
@@ -181,7 +181,7 @@
         <Setter Property="MaxDropDownHeight" Value="90" />
         <Setter Property="MinimumPrefixLength" Value="1" />
         <Setter Property="Focusable" Value="False" />
-        <Setter Property="IsTextCompletionEnabled" Value="True" />
+        <Setter Property="IsTextCompletionEnabled" Value="False" />
```

Comment:
In the case of `RestrictInputToSuggestions="True"` can't we have the old behavior? Because the issue mentioned above is not possible to happen and i.e. in Recover Wallet case the user still has the possibility to enter the recovery words superfast.

@danwalmsley What do you think?

```suggestion
            <Setter Property="IsTextCompletionEnabled" Value="{Binding Path=RestrictInputToSuggestions, RelativeSource={RelativeSource AncestorType={x:Type c:TagsBox}}}" />

```

Reply:
Yes i agree with this one, Recovery words are fixed anyway and should be case insensitive.